### PR TITLE
Add runbook for VirtControllerRESTErrorsBurst alert

### PR DESCRIPTION
**Pull request by sradco**
Signed-off-by: Shirly Radco <sradco@redhat.com>

### CODE REVIEW

**Comment by sradco** [comment]
@iholder-redhat Hi, Can you please review and add the command the user should run for identifying the issue and mitigating it?

**Comment by sradco** [comment]
@iholder-redhat Hi, Can you please review and add the command the user should run for identifying the issue and mitigating it?

**Comment by iholder-redhat** [comment]
Hey @sradco,

As we talked privately:
Honestly, I don't think that a user is able to identify or mitigate high REST failures from one of our core components (virt-handler / virt-launcher). There are several reasons for that.

The first one being that it can be caused because a very large range of problems. This is a super generic indication of something being wrong, but without a deeper debug on a specific case it's hard to mitigate it

Do you have real cases when this alert popped up?

Another reason is that it's one of Kubevirt core components. The chance of them being screwed up because of the user is very low, and even if it does happen it's probably due to our bug, not to the user's behavior. In other words - the user is not expected to being able to screw up virt-handler or virt-controller.

Maybe someone else can assist here, but I'm not sure there's an easy general way of dealing with such problem.

P.S:
This comment applies to the same question you've asked in the other similar PRs: https://github.com/kubevirt/monitoring/pull/30, https://github.com/kubevirt/monitoring/pull/34, https://github.com/kubevirt/monitoring/pull/35,

**Comment by sradco** [comment]
> Hey @sradco,
> 
> As we talked privately: Honestly, I don't think that a user is able to identify or mitigate high REST failures from one of our core components (virt-handler / virt-launcher). There are several reasons for that.
> 
> The first one being that it can be caused because a very large range of problems. This is a super generic indication of something being wrong, but without a deeper debug on a specific case it's hard to mitigate it
> 
> Do you have real cases when this alert popped up?
> 
> Another reason is that it's one of Kubevirt core components. The chance of them being screwed up because of the user is very low, and even if it does happen it's probably due to our bug, not to the user's behavior. In other words - the user is not expected to being able to screw up virt-handler or virt-controller.
> 
> Maybe someone else can assist here, but I'm not sure there's an easy general way of dealing with such problem.
> 
> P.S: This comment applies to the same question you've asked in the other similar PRs: #30, #34, #35, #54

@iholder-redhat Hi, I dont have an issue with the fact that there can be a lot of reasons as to why the alerts can fire. The only issue is that what currently is written in the mitigation part is not mitigation. Please move it to the diagnosis section and ask the user to open a ticket with X, Y, Z attached with Instructions to what they should attach and how to produce it.

File: docs/runbooks/VirtControllerRESTErrorsBurst.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+## Diagnosis
+
+There exist two common types of errors:
+- The apiserver is overloaded and we run into timeouts.
+  Issues like this can be identified by checking the apiserver metrics and looking at its response times, overall calls, …
+- The virt-controller pod can’t reach the apiserver. Common issues are: dns issues on the node, networking connectivity issues
+
+## Mitigation
+
+Check virt-controller logs to identify if it can’t connect to the apiserver at all. If so, delete the pod to force a restart. In this case the issue is normally related to DNS or CNI issues outside of the scope of kubevirt.
```
Comment (sradco) [inline docs/runbooks/VirtControllerRESTErrorsBurst.md:20]: @iholder-redhat How do I know which pod to restart? (There are 3)
I would also add something like:
"Please open an issue and attach the artifacts gathered in the Diagnosis section." 
and add the commands to run for gathering the artifacts to the diagnosis section.

**Pull request merged**